aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -282,65 +275,58 @@ dotnet ef database update
 
 Examine the *Migrations/{timestamp}_InitialCreate.cs* migration file:
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Migrations/20190805165915_InitialCreate.cs?name=snippet)]
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie5/Migrations/20201107014924_InitialCreate.cs?name=snippet)]
 
-The `Up` method creates the Movie table and configures `Id` as the primary key. The `Down` method reverts the schema changes made by the `Up` migration.
+In the preceding code:
+
+* `InitialCreate.Up` creates the Movie table and configures `Id` as the primary key.
+* `InitialCreate.Down` reverts the schema changes made by the `Up` migration.
 
 <a name="test"></a>
 
 ## Test the app
 
-* Run the app and click the **Movie App** link.
+Run the app and select the **Movie App** link.
 
-  If you get an exception similar to one of the following:
+If you get an exception similar to the following, you may have missed the [migrations step](#migration):
 
 # [Visual Studio](#tab/visual-studio)
 
-  ```console
-  SqlException: Cannot open database "MvcMovieContext-1" requested by the login. The login failed.
-  ```
+```console
+SqlException: Cannot open database "MvcMovieContext-1" requested by the login. The login failed.
+```
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
 
-  ```console
-  SqliteException: SQLite Error 1: 'no such table: Movie'.
-  ```
+```console
+SqliteException: SQLite Error 1: 'no such table: Movie'.
+```
 
 ---
-  You probably missed the [migrations step](#migration).
 
-* Test the **Create** page. Enter and submit data.
+Test the **Create** page. Enter and submit data.
 
-  > [!NOTE]
-  > You may not be able to enter decimal commas in the `Price` field. To support [jQuery validation](https://jqueryvalidation.org/) for non-English locales that use a comma (",") for a decimal point and for non US-English date formats, the app must be globalized. For globalization instructions, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/4076#issuecomment-326590420).
+> [!NOTE]
+> You may not be able to enter decimal commas in the `Price` field. To support [jQuery validation](https://jqueryvalidation.org/) for non-English locales that use a comma (",") for a decimal point and for non US-English date formats, the app must be globalized. For globalization instructions, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/4076#issuecomment-326590420).
 
-* Test the **Edit**, **Details**, and **Delete** pages.
+Test the **Edit**, **Details**, and **Delete** pages.
 
 ## Dependency injection in the controller
```

Comment:
Move this up under Examine the generated database context registration where you're talking about it.